Regex added for 3.16.x support

vigneshk-fd opened the pull request:
- Added equivalent regex to handle cases where the ember build cache paths can be different (ex: 3.4.x vs 3.16.x versions)

cibernox committed:
Update ember-cli-babel & broccoli-babel-transpiler

cibernox committed:
Use ember-cli-babel for transpiration

cibernox committed:
Fix tests

villander committed:
javascript optimization for loop with cache length

Commit:
Improve devex for acceptance test recursive installation

Commit:
Attempt to upgrade to ember 2.18.0

Commit:
Get tests running

Commit:
Fix fastboot tests and drop node 4

Commit:
0.7.0

ming-codes committed:
Upgrade uglifyjs

NullVoxPopuli committed:
this change works locally for emberclear

scottkidder committed:
chore: update package-lock.json

scottkidder committed:
tests: fix minifyJS test after uglifyjs upgrade

Commit:
Update to broccolijs/broccoli-test-helper

Commit:
Upgrade ember and test dependencies

Commit:
Update general node dependencies

Commit:
Upgrade broccoli-rollup

Commit:
Validate version strategy and fall back when invalid/not provided

Commit:
0.7.1

stukalin committed:
added the regular expression for windows paths

stukalin committed:
comment for the regex

Commit:
Update dep graph

Commit:
Update gems

Commit:
Merge branch 'villander-master'

* villander-master: javascript optimization for loop with cache length

stukalin committed:
better comment for the regex

Commit:
0.7.2

vigneshk-fd committed:
Regex added for 3.16.x support

# Added equivalent regex to handle cases where the ember build cache paths can be different (ex: 3.4.x vs 3.16.x versions)

vigneshk-fd committed:
Remove unused cookie name variable

vishnubalajikm commented on lib/service-worker-builder.js:
```diff
@@ -89,7 +89,13 @@ module.exports = class ServiceWorkerBuilder {
+        '**/ember-service-worker/**/*.js',
+        // same as above, but added equivalent regex to handle cases 
+        // where the ember build cache paths can be different (ex: 3.4.x vs 3.16.x versions)
+        /^(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.{1,2}(?:\/|$)).)*?)\/)ember-service-worker(?:\/(?!\.{1,2}(?:\/|$))(?:(?:(?!(?:^|\/)\.{1,2}(?:\/|$)).)*?)\/|\/|$)(?!\.{1,2}(?:\/|$))(?=.)[^/]*?\.js)$/
```
Can we pass this regex from the environment options ?

vigneshk-fd committed:
Took regex from master branch

# Took a regex change alone from master
# Reverse merge with master branch - Not Done.

vigneshk-fd committed:
Merge branch 'master' into include-file-path-regex-addition

vigneshk-fd committed:
Revert "Merge branch 'master' into include-file-path-regex-addition"

This reverts commit bf6357a9359dd958dcc421aa3f313935d79b4aab, reversing
changes made to 6311101eb7f2edfd618e795ec463aa31683b756d.

vigneshk-fd committed:
Update service-worker-builder.js
